code/obj/item/rcd.dm

```diff
@@ -64,14 +66,16 @@ Broken RCD + Effects
 
 	var/matter_create_floor = 1
 	var/matter_create_wall = 2
-	var/matter_create_wall_girder = 1
+	var/matter_create_wall_girder = 2
```

Comment:
No sneaking changes outside the scope of the PR in, please!

Reply:
Huh, didn't mean to change that.